nodes/dat-chunk/create-dat-node.js

```diff
@@ -0,0 +1,65 @@
+var fs = require('fs')
+var url = require('url')
+var path = require('path')
+var Dat = require('dat-node')
+var smalltalk = require('smalltalk')
+var mirror = require('mirror-folder')
+var ram = require('random-access-memory')
+
+module.exports = function (collection, index, context, fileObject, cb) {
+  smalltalk.prompt('Enter Dat url', '', 'dat://').then(function (value) {
+    var datDetails = url.parse(value)
+    var datHash = datDetails.hostname
+
+    var folderPath = fileObject.resolvePath(datHash)
+    fetchDat(datHash, folderPath, function (err) {
+      var path = fileObject.resolvePath(datHash + '/index.json')
+      var externalDescriptor  = {
+        node: 'Node 1',
```

Comment:
Like I mentioned in the comment above, you might want to include some of these params (particularly shape) in the export itself.

Reply:
Ok, thats a good idea. I need to start thinking of how to add meta to the chunk so publishers can help people find them. I think for the exported chunk, I will add a property to the index.json called "published". So it can contain published metadata that can be used by the externalDescriptor

so the file might look like this at the top
e9569d6ff35812c3269e55b3be144512ac6bad1f3ec8510fd389afd0fdcd202a/index.json

```
{
  "published": {
    "name": "large drumkit",
    "publisher": "Ryan Ramage",
    "tags": ["drumkit", "samples"], 
    "shape": [8, 8]
    "color": [251, 141, 197],
    "paramValues": { "dist": 0, "delayinfit": 0 },
    "volume": 1.4428
  },
  slots: {
    ...
```

Later we can start indexing this info into a db and make a search that can help people find stuff. 


I think I want to add a 'publish' button to the tab that appears when you edit a chunk.